### ISSUE TITLE
ONYX-1522: Variables not found when uploading the questionnaire InHome_Id

### ISSUE THREAD

**Issue by ymarcon** [obiba/onyx#1393]
Jira issue originally created by user @emorency:

I uploaded the questionnaire InHome_Id locally and I had the exception (to resolve this problem, I had to create a new variable locally and then the variables were ALL 'visible'):
SEVERE: Error while resolving variable values in: So your age is $('age*1*answer')? Is that correct?
org.obiba.magma.NoSuchVariableException: No such variable 'age*1_answer' in table 'InHome*Id'
	at org.obiba.magma.support.AbstractValueTable.getVariableValueSource(AbstractValueTable.java:127)
	at org.obiba.onyx.core.data.VariableDataSource.getVariableValueSource(VariableDataSource.java:131)
	at org.obiba.onyx.core.data.VariableDataSource.getValue(VariableDataSource.java:85)
	at org.obiba.onyx.quartz.core.wicket.model.QuestionnaireStringResourceModel.resolveVariableValuesInMessage(QuestionnaireStringResourceModel.java:131)
	at org.obiba.onyx.quartz.core.wicket.model.QuestionnaireStringResourceModel.load(QuestionnaireStringResourceModel.java:115)
	at org.obiba.onyx.quartz.core.wicket.model.QuestionnaireStringResourceModel.load(QuestionnaireStringResourceModel.java:31)
	at org.apache.wicket.model.LoadableDetachableModel.getObject(LoadableDetachableModel.java:120)
	at org.apache.wicket.Component.getDefaultModelObject(Component.java:1724)
	at org.apache.wicket.Component.getDefaultModelObjectAsString(Component.java:1743)
	at org.apache.wicket.markup.html.basic.Label.onComponentTagBody(Label.java:114)
	at org.apache.wicket.Component.renderComponent(Component.java:2690)
	at org.apache.wicket.markup.html.WebComponent.onRender(WebComponent.java:62)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1441)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1604)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:697)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:114)
	at org.apache.wicket.Component.renderComponent(Component.java:2690)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1539)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1441)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1604)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1528)
	at org.apache.wicket.Component.renderComponent(Component.java:2690)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1539)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.markup.repeater.AbstractRepeater.renderChild(AbstractRepeater.java:122)
	at org.apache.wicket.markup.repeater.AbstractRepeater.onRender(AbstractRepeater.java:103)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1441)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1604)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:697)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:114)
	at org.apache.wicket.Component.renderComponent(Component.java:2690)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1539)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1441)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1604)
	at org.apache.wicket.MarkupContainer.renderAssociatedMarkup(MarkupContainer.java:697)
	at org.apache.wicket.markup.html.panel.Panel.onComponentTagBody(Panel.java:114)
	at org.apache.wicket.Component.renderComponent(Component.java:2690)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1539)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1441)
	at org.apache.wicket.MarkupContainer.renderComponentTagBody(MarkupContainer.java:1604)
	at org.apache.wicket.MarkupContainer.onComponentTagBody(MarkupContainer.java:1528)
	at org.apache.wicket.markup.html.form.Form.onComponentTagBody(Form.java:2012)
	at org.apache.wicket.Component.renderComponent(Component.java:2690)
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1539)
	at org.apache.wicket.Component.render(Component.java:2521)
	at org.apache.wicket.Component.renderComponent(Component.java:2631)
	at org.apache.wicket.ajax.AjaxRequestTarget.respondComponent(AjaxRequestTarget.java:901)
	at org.apache.wicket.ajax.AjaxRequestTarget.respondComponents(AjaxRequestTarget.java:681)
	at org.apache.wicket.ajax.AjaxRequestTarget.respond(AjaxRequestTarget.java:591)
	at org.apache.wicket.request.AbstractRequestCycleProcessor.respond(AbstractRequestCycleProcessor.java:105)
	at org.apache.wicket.RequestCycle.processEventsAndRespond(RequestCycle.java:1258)
	at org.apache.wicket.RequestCycle.step(RequestCycle.java:1329)
	at org.apache.wicket.RequestCycle.steps(RequestCycle.java:1436)
	at org.apache.wicket.RequestCycle.request(RequestCycle.java:545)
	at org.apache.wicket.protocol.http.WicketFilter.doGet(WicketFilter.java:486)
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:319)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1322)
	at org.springframework.orm.hibernate3.support.OpenSessionInViewFilter.doFilterInternal(OpenSessionInViewFilter.java:198)
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:76)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1322)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:473)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:119)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:516)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:226)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:929)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:403)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:184)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:864)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:117)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:247)
	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:151)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:114)
	at org.eclipse.jetty.server.Server.handle(Server.java:352)
	at org.eclipse.jetty.server.HttpConnection.handleRequest(HttpConnection.java:596)
	at org.eclipse.jetty.server.HttpConnection$RequestHandler.content(HttpConnection.java:1068)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:805)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:218)
	at org.eclipse.jetty.server.HttpConnection.handle(HttpConnection.java:426)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:508)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.access$000(SelectChannelEndPoint.java:34)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:40)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:451)
	at java.lang.Thread.run(Thread.java:679)
